src/helpers/validateErrorHandler.js

```diff
@@ -0,0 +1,18 @@
+import ResponseService from '../services/response.service';
+
+const validateErrorHandler = (schema, body, res, next) => {
```

Comment:
This is a helper function which brings all errors from joi at once. I saw we all had to copy it and that would cause duplicates as code grows. I call it to generate the error without rewrite it.